go-controller/pkg/ovn/master.go

```diff
@@ -406,7 +403,8 @@ func (oc *Controller) SetupMaster(masterNodeName string, existingNodeNames []str
 	}
 
 	// Create a cluster-wide port group that all logical switch ports are part of
-	oc.clusterPortGroupUUID, err = createPortGroup(oc.ovnNBClient, clusterPortGroupName, clusterPortGroupName)
+	pg := libovsdbops.BuildPortGroup(types.ClusterPortGroupName, types.ClusterPortGroupName, nil, nil)
```

Comment:
Right, I think that just passing a "stub" PortGroup struct around is way cleaner. I'm not a fan of lots of parameters, especially when nil-able.

Reply:
I know that for PG the function looks simple enough that might not be needed but I am trying to set a pattern given that for other types like ACLs they do come in handy. So the approach is to be as opaque as possible with the structs out of `libovsdbops` to try to hide any logic that goes within them now and in the future.